sql/ha_partition.cc

```diff
@@ -5663,6 +5663,19 @@ int ha_partition::read_range_next()
   DBUG_RETURN(handle_unordered_next(table->record[0], eq_range));
 }
 
+bool ha_partition::has_gap_locks()
```

Comment:
Yeah, well I would also hope that by then we have gap lock support and can drop this entirely, but then, this still might make sense to keep around.